File: oas/tiktok/RockWallClimber.java
package oas.tiktok;

public class RockWallClimber {
    public static int jumps(int bigJump, int height){
        int count = 0;

        while (height > 0){
            if (height >= bigJump){
                height -= bigJump;
                count++;
            }
            else{
                height -= 1;
                count++;
            }
        }
        return count;
    }

    public static void main(String[] args) {
        int height = 10000;
        int bigJump = 1;

        System.out.println(jumps(bigJump, height));
    }
}

File: python/oas/tiktok/rock_wall_climber.py
from typing import Counter


class rockWallClimber:
    def jumps(self, flag_height: int, big_jump: int) -> int:
        count = 0
        
        while flag_height > 0:
            if flag_height >= big_jump:
                flag_height -= big_jump
                count += 1
            else:
                flag_height -= 1
                count += 1
        return count


if __name__ == "__main__":
    print(rockWallClimber().jumps(8, 3))
        